projects/kit/components/input-phone-international/interfaces/country.ts

```diff
@@ -1,4 +1,13 @@
 export interface Country {
```

Comment:
Let's deprecate the whole interface, there's no need for an object with single key in the future.

Reply:
done